examples/02_decoding/plot_miyawaki_encoding.py

```diff
@@ -0,0 +1,259 @@
+
+from nilearn.image import threshold_img, new_img_like
+from nilearn.masking import unmask
+
+cut_score = np.array(scores).mean(axis=0)
+cut_score[cut_score < 0] = 0
+
+# bring the scores into the shape of the background brain
+score_map_img = new_img_like(dataset.background,
+                         unmask(cut_score, dataset.mask).get_data())
+
+thresholded_score_map_img = threshold_img(score_map_img, threshold=1e-6)
+
+##############################################################################
+# Plotting the statistical map on a background brain, we mark four voxels
+# which we will inspect more closely later on.
+from nilearn.plotting import plot_stat_map
+from nilearn.image.resampling import coord_transform
+
+def index_to_xy_coord(x, y, z=10):
+    '''Transforms data index to coordinates of the background + offset'''
+    coords = coord_transform(x, y, z,
+                             affine=thresholded_score_map_img.get_affine())
+    return np.array(coords)[np.newaxis, :] + np.array([0, 1, 0])
+
+
+xy_indices_of_special_voxels = [(30, 10), (32, 10), (31, 9), (31, 10)]
+
+display = plot_stat_map(thresholded_score_map_img, bg_img=dataset.background,
+                        cut_coords=[-8], display_mode='z', aspect=1.25,
+                        title='Explained variance per voxel')
+
+# creating a marker for each voxel and adding it to the statistical map
+
+for i, (x, y) in enumerate(xy_indices_of_special_voxels):
+    display.add_markers(index_to_xy_coord(x, y),
+                        edgecolor=['b', 'r', 'magenta', 'g'][i],
+                        marker_size=150, marker='s',
+                        facecolor='none', lw=4.5)
+
+
+# re-set figure size after construction so colorbar gets rescaled too
+fig = plt.gcf()
+fig.set_size_inches(12, 12)
+
+
+##############################################################################
+# Estimating receptive fields
+# ---------------------------
+# Now we take a closer look at the receptive fields of the four marked voxels.
+# A voxel's `receptive field <http://en.wikipedia.org/wiki/Receptive_field>`_
+# is the region of a stimulus (like an image) where the presence of an object,
+# like a white instead of a black pixel, results in a change in activity
+# in the voxel. In our case the receptive field is just the vector of 100
+# regression  coefficients (one for each pixel) reshaped into the 10x10
+# form of the original images. Some voxels are receptive to only very few
+# pixels, so we use `Lasso regression
+# <http://en.wikipedia.org/wiki/Lasso_(statistics)>`_ to estimate a sparse
+# set of regression coefficients.
+
+from sklearn.linear_model import LassoLarsCV
+
+# automatically estimate the sparsity by cross-validation
+lasso = LassoLarsCV(max_iter=10)
+
+# Mark the same pixel in each receptive field
+marked_pixel = (4, 2)
+
+from matplotlib import gridspec
+from matplotlib.patches import Rectangle
+
+fig = plt.figure(figsize=(12, 12))
```

Comment:
I wonder if that figure isn't too tall: the aspect ratio should probably be around 3/2. Right now, it has a lot of whitespace at the bottom.